src/lib/download.ts

```diff
@@ -22,6 +30,9 @@ export function downloadBlob(blob: Blob, fileName: string) {
   anchor.click()
```

Comment:
I think we should use download apis https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads

Reply:
Or use `writeFile` method from `electronOnly`

Reply:
I didn't really write this method, but will need more time to investigate this downloads API, is this something we could use from electron API? Yes I see, for blob it don't matter since we could just write it to disk? Will update it later today.

Reply:
So I investigated this further. For download APIs - that's extensions? In this case electron supports some, but it seems a bit complicated setting that up, do you have any examples of this? did you try it?

As for writeFile, I could call something like this:

`
await writeFile(fileName, Buffer.from(await blob.arrayBuffer()))
`

similar is done for attachment writing. But this involves promises and a drawback that we are saving it with filename only - location should then be fetched from user somehow (folder where to save the item, as we have for storage location selection boxes).

This is currently done with ".click()" when user selectes the location, in that location/filename the blob is saved/downloaded.

So can you inspect this one more time, and give me the specification, which method would be best, so we can change it?

Reply:
I guess we can merge this now. I'll investigate more later!